app/models/user.rb

```diff
@@ -83,4 +83,16 @@ def organisation_name
       (is_computacenter? && 'Computacenter') || \
       (is_support? && 'DfE Support')
   end
+
+  def first_name
+    (full_name || '').strip.split(' ').first.to_s
+  end
+
+  def last_name
+    (full_name || '').strip.split(' ').last.to_s
+  end
+
+  def derived_responsible_body
+    responsible_body || school.responsible_body
```

Comment:
Since we're defining this for all users (who can have neither responsible body nor school e.g. support users), we should guard for this:

```suggestion
    responsible_body || school&.responsible_body
```

Reply:
done